website/content/en/preview/upgrade/upgrade-guide.md

```diff
@@ -8,120 +8,160 @@ description: >
+    kubectl apply -f https://raw.githubusercontent.com/chrisnegus/karpenter/docs-v1beta1-migrate/pkg/apis/crds/karpenter.sh_nodeclaims.yaml
+    kubectl apply -f https://raw.githubusercontent.com/chrisnegus/karpenter/docs-v1beta1-migrate/pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml
+    ```
 
-In general, you can reapply the CRDs in the `crds` directory of the Karpenter helm chart:
+6. Upgrade Karpenter to the new version:
+
+    ```bash
+    helm registry logout public.ecr.aws
+    
+    helm upgrade --install karpenter oci://public.ecr.aws/karpenter/karpenter --version ${KARPENTER_VERSION} --namespace karpenter --create-namespace \
+      --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"=${KARPENTER_IAM_ROLE_ARN} \
+      --set settings.aws.clusterName=${CLUSTER_NAME} \
+      --set settings.aws.interruptionQueueName=${CLUSTER_NAME} \
+      --set controller.resources.requests.cpu=1 \
+      --set controller.resources.requests.memory=1Gi \
+      --set controller.resources.limits.cpu=1 \
+      --set controller.resources.limits.memory=1Gi \
+      --wait
+    ```
 
-```shell
-kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef >}}pkg/apis/crds/karpenter.sh_provisioners.yaml
-kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef >}}pkg/apis/crds/karpenter.sh_machines.yaml
-kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef >}}pkg/apis/crds/karpenter.k8s.aws_awsnodetemplates.yaml
-```
+7. Convert each AWSNodeTemplate to an EC2NodeClass. To convert your v1alpha Karpenter manifests to v1beta1, you can either manually apply changes to API components or use the [Karpenter conversion tool](https://github.com/aws/karpenter/tree/main/tools/karpenter-convert/README).
+   See the [AWSNodeTemplate to EC2NodeClass]({{< relref "v1beta1-reference#awsnodetemplate-to-ec2nodeclass" >}}) section of the Karpenter Upgrade Reference for details on how to update to Karpenter AWSNodeTemplate objects. Here is an example of how to use the `karpenter-convert` CLI to convert an AWSNodeTemplate file to a EC2NodeClass file:
 
-### How Do We Break Incompatibility?
+    ```bash
+    karpenter-convert -f awsnodetemplate.yaml > ec2nodeclass.yaml
+    ```
 
-When there is a breaking change we will:
+8. Edit the converted EC2NodeClass file manually:
 
-* Increment the minor version when in major version 0
-* Add a permanent separate section named `upgrading to vx.y.z+` under [released upgrade notes](#released-upgrade-notes)
-  clearly explaining the breaking change and what needs to be done on the user side to ensure a safe upgrade
-* Add the sentence “This is a breaking change, please refer to the above link for upgrade instructions” to the top of the release notes and in all our announcements
+   * Specify your AWS role where there is a `your AWS role Here` placeholder.
```

Comment:
As part of this PR https://github.com/aws/karpenter/pull/4833
the placeholder now is `$KARPENTER_NODE_ROLE`